src/helpers/ApiCallHelper.tsx

```diff
@@ -1,7 +1,28 @@
+const baseURL = 'https://mobile-api-softwire2.lner.co.uk/v1';
+
 export const fetchStations = () => {
     return fetch('https://mobile-api-softwire2.lner.co.uk/v1/stations', {
         headers: {
             'X-API-KEY': `${process.env.REACT_APP_X_API_KEY}`,
         },
     });
 };
+
+type journey = {

```

Comment:
This looks like it's the response for the fares endpoint, I'd probably refactor this out to its own file and name it a bit more explicitly, e.g. JourneySearchResponse or JourneyResponse or FaresResponse.

Reply:
Good job doing this nice and early - it's good practice to have models of your API responses so you can verify them and work with them nicely. Don't worry too much about fully mapping out the fares response as it it fairly meaty, but if you want you can check out their example response and schema here: http://lner-build-docker.zoo.lan/swagger/#/retail/fare_search